cmd/version.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+
+	goVersion "github.com/christopherhein/go-version"
+	"github.com/spf13/cobra"
+)
+
+var (
+	shortened = false
+	version   string
+	commit    string
+	date      string
+)
+
+func versionCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Version will output the current build information",
+		Long:  ``,
+		Run: func(_ *cobra.Command, _ []string) {
```

Comment:
```suggestion
		RunE: func(*cobra.Command, []string) error {
```

- With cobra always use the version that allows us to return an error.
- When you ignore both arguments to a function, you can omit all of them (you don't even need underscores); when you ignore some and use some you must put names for all of them (where `_` is a valid "name").